File: src/github.com/heartszhang/feed/feedsource.go
package feed

type FeedSource struct {
	Name        string   `json:"name,omitempty" bson:"name,omitempty"`
	Uri         string   `json:"uri,omitempty" bson:"uri,omitempty"`         // rss/atom url
	Period      int64    `json:"period" bson:"period"`                       // minutes
	Update      int64    `json:"update" bson:"update"`                       // unixtime
	Type        uint     `json:"type" bson:"type"`                           // feed_type...
	WebSite     string   `json:"website,omitempty" bson:"website,omitempty"` // home
	Description string   `json:"description,omitempty" bson:"description,omitempty"`
	Logo        string   `json:"logo,omitempty" bson:"logo,omitempty"`
	Hub         string   `json:"hub,omitempty" bson:"hub,omitempty"`
	Tags        []string `json:"tags,omitempty" bson:"tags,omitempty"`
}

File: famousfront/datamodels/FeedSource.cs
using System.Runtime.Serialization;

namespace famousfront.datamodels
{
  [DataContract]
  internal class FeedSource
  {
    [DataMember(EmitDefaultValue = false)]
    public string name
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public string uri
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public string local
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public uint period
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public ulong deadline
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public uint type
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public int subscribe_state
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public bool enable_proxy
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public ulong update
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public string website
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public string[] tags
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public string[] categories
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public int unreaded
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public FeedMedia media
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public string description
    {
      get;set;
    }
    [DataMember(EmitDefaultValue = false)]
    public string logo
    {
      get;
      set;
    }
  }
}